src/submission/validation-clinical/followUp.ts

```diff
@@ -81,6 +89,31 @@ export const validate = async (
   return { errors };
 };
 
+function checkTreatmentTimeConflict(
```

Comment:
is there a need to validate when the treatment file gets updated (users can update the treatment_start_interval)?